packages/cln_plugin/lib/src/rpcmethods/manifest/getmanifest.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:cln_plugin/src/rpcmethods/abstractrpcmethod.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/option.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/feature.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/notification.dart';
+
+class ManifestMethod extends AbstractRpcMethod {
+  late List<AbstractRpcMethod> _rpcMethods;
+  late List<String> _subscriptions;
+  late List<Option> _options;
+  late List<String> _hooks;
+  late Feature _features;
+  late bool _dynamic = false;
+  late Set<Notification> _notifications;
```

Comment:
```suggestion
  late List<AbstractRpcMethod> _rpcMethods;
  late List<String> _subscriptions;
  late List<Option> _options;
  late List<String> _hooks;
  late Feature _features;
  late bool _dynamic = false;
  late Set<Notification> _notifications;
```

All public and we remove all the get and setter